### DISCUSSION TITLE
Charging via pins

### DISCUSSION THREAD

**Discussion by DomNomNom** [bitcraze/discussions#1090] [Q&A] [answered]
I want to make a setup where my drone can charge via the extension board pins. This has to be possible somehow given that there's the Qi charging board. I'd like to keep my drone's weight low which is why I'm electing to not go with that extension board.

Is the following circuit diagram sufficient to charge the drone?
![image](https://github.com/bitcraze/discussions/assets/888068/c3a7c365-fa96-496e-9efe-2c3f2ddca405)
is the WKUP pin connection necessary?
Do I need to have capacitors between VCC and ground?

in terms of physical design, I'm planning on having longer pins only on those three holes. then a 3D printed form that aligns the drone that comes in guided by a LH positioning system. then have contactors like little leaf springs, kinda like on limit switches

**Comment by ataffanel** [comment, marked as answer]
Hi,

The wkup connection should not be necessary. Only GND and VCC are required. We have been experimenting with contact charging and we have found out that pogo-pins are requires if you plan to land autonomously on the charger. Spring on the charger side could works as well.

Our most successful design so far is to have 2 concentric circular tracks for the charging pad and pogo pins alingned to hit the 2 radius on the Crazyflie side, this way, only x/y has to be alligned, the yaw  does not have to.
